lib/fhir/definitions/dstu2/search-parameters.json

```diff
@@ -9849,7 +9849,7 @@
         "base": "DocumentReference",
         "type": "token",
         "description": "Master Version Specific Identifier",
-        "xpath": "f:DocumentReference/f:masterIdentifier",
+        "xpath": "f:DocumentReference/f:masterIdentifier | f:DocumentReference/f:identifier",
```

Comment:
With these changes I had to update the FHIR spec as it didn't correctly include all paths when a param has multiple paths. There may be more of these errors in DSTU2. It seems to be ok in STU3 though.